src/languageFeatures.ts

```diff
@@ -1204,7 +1213,7 @@ export class RenameAdapter extends Adapter implements languages.RenameProvider {
 		const edits: languages.WorkspaceTextEdit[] = [];
 		for (const renameLocation of renameLocations) {
 			const resource = Uri.parse(renameLocation.fileName);
-			const model = editor.getModel(resource);
```

Comment:
If a extraLib file reference a  type  Foo of this file, then trigger find reference of  Foo in this  file won't list that extraLib file as a result

since right now extraLib file won't auto crate a monaco model, This PR will fix this